internal/rbd/encryption.go

```diff
@@ -64,15 +64,15 @@ const (
 func (ri *rbdImage) checkRbdImageEncrypted(ctx context.Context) (rbdEncryptionState, error) {
 	value, err := ri.GetMetadata(encryptionMetaKey)
 	if errors.Is(err, librbd.ErrNotFound) {
-		util.DebugLog(ctx, "image %s encrypted state not set", ri.String())
```

Comment:
`ri`, `rv`, `cp` in `encryption.go` are all pointers, can be inferred from the method definition in which it is like 
`@@ -97,14 +97,14 @@ func (ri *rbdImage) setupEncryption(ctx context.Context) error {`